apps/alpha-test-app/src/App.vue

```diff
@@ -30,7 +29,6 @@ const navItems = [
 ] satisfies OnyxNavItemProps[];
 
 const { store: colorScheme } = useColorMode();
-const isColorSchemeDialogOpen = ref(false);
```

Comment:
was this an accident?
or did you just forget to remove `<OnyxColorSchemeDialog` in line 85?

Pipeline fails because of that:
https://github.com/SchwarzIT/onyx/actions/runs/9747211600/job/26899485234?pr=1465